about.html

```diff
@@ -0,0 +1,140 @@
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Back to Home Page</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The Burning Circle Flight Academy</h1>
+      <div class="first-section-para">
+        <p>
+          The Burining Circle Flight Academy airshow is one of Zambia's first airshows 
+          to be hosted by a privately owned aviation institution. The whole essence of 
+          this event is to connect aerospace industry proffessionals across Africa and
+          the world at Large. This is also a great opportunity for youths interested in
+          aviation and becoming Pilots to learn from the best Pilots and know what it takes
+          to become a world class pilot. 
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        
+        </p>
+        <p class="email-address">
+          burningcircleflightacademy@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> The Burning Circle Flight Academy Airshow</h2>
+      <hr>
+      <p class="mobile-para">This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+      <p class='desktop-para'> This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>Past Airshow events</h2>
+      <hr>
+      <p>Take a look at the past airshows we attended and what inspired us to host our own Airshow event as a Pilot training institution.</p>
+      <div class="pics">
+      <img src="images/airshow1.jpeg" alt="Dubai airshow2019">
+      <img src="images/airshow2.jpeg" alt="Dubai airshow2020">
```

Comment:
- [ ] the two images in the last section in the `about` page should have a color mask along with text centered over them, also they need to be the exact same size on both screen sizes, please add them so your project looks perfect!

Reply:
done